src/BoilerPlate/App.cpp

```diff
@@ -561,4 +653,13 @@ namespace Engine
 			m_bullets[i]->Render();
 		}
 	}
-}
\ No newline at end of file
+
+	void App::WarpEntities(float newHeight, float newWidth)
+	{
+		for (int i = 0; i < m_entities.size(); i++)
+		{
+			m_entities[i]->OnWindowResize(newHeight, newWidth);
```

Comment:
This is ok, but I'd have a parent object that handles this and then make the entities query the size.